Client/metrics.yaml

```diff
@@ -1834,6 +1820,74 @@ widget:
       - fx-ios-data-stewards@mozilla.com
     expires: "2022-07-01"
 
+# Wallpaper metrics
+wallpaper_analytics:
+  toggle_logo_wallpaper_button:
+    type: event
+    description: |
+      Recorded when user toggles the button which enables
+      or disables the ability to toggle wallpapers on the
+      homepage from the logo.
+    extra_keys:
+      changed_to:
+        type: string
+        description: |
+          The value the switch was set to.
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXIOS-3487
+    data_reviews:
+      - https://mozilla-hub.atlassian.net/browse/FXIOS-3487
+      - https://github.com/mozilla-mobile/firefox-ios/pull/9673
```

Comment:
Lets update this with this PR link

Reply:
Updating it in the commit where I'm addressing some of Laurie's things. :)